fixtures/src/components/controlled/checkbox-group.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { Checkbox, CheckboxGroup } from 'carbon-react/lib/__experimental__/components/checkbox';
+import { LogConsumer } from '../log';
+
+const ControlledCheckbox = () => {
+  const [checked, setChecked] = useState({
+    controlled_checkbox_group_default_two: true
```

Comment:
I think it'd be good to initialise all the fields, so we have a clear listing of all the fields that are used:
```js
controlled_checkbox_group_one: false,
controlled_checkbox_group_two: false,
controlled_checkbox_group_three: false,
controlled_checkbox_group_default_one: false,
controlled_checkbox_group_default_two: true,
controlled_checkbox_group_default_three: false
```

Reply:
Thanks for adding this in 04db74c :white_check_mark: